seedlink_plotter/seedlink_plotter.py

```diff
@@ -474,9 +497,61 @@ def get_events(self):
         with self.lock:
             start = min([tr.stats.starttime for tr in self.stream])
             end = max([tr.stats.endtime for tr in self.stream])
-        neries_emsc = Client("EMSC")
-        events = neries_emsc.get_events(starttime=start, endtime=end,
-                                          minmagnitude=self.args.events)
+        
+        events = Catalog()
+        
+         # get the coordinats of the station
+        splitted_ids = [x.split(".") for x in self.args.ids]  # use seedlink id to get network, station, location and channel
+        for client in ["iris-federator","eida-routing"]:
```

Comment:
PEP8 once more, space after `,`

Reply:
Also.. I don't like this guessing what client should be used.. and the usage of federated clients. Is there a problem with only allowing access to individual FDSN nodes? I don't think federators are used much for events anyways. Federators are mostly for waveform data.

I think we should only use plain FDSN clients.

Reply:
Ah, I see you're trying to get station metadata.. I think the user should instead have to specify in the config file where the metadata is coming from. For now I think it's OK to simply use some config variable that specifies a FDSN node as station metadata source.

I think it's better to have to specify this instead of requests to obscure federated clients that might time out or don't have the data anyway (e.g. you won't find our ringlaser on IRIS or EIDA..)